adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -124,29 +124,27 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
 
   val stringency = ValidationStringency.valueOf(args.stringency)
 
-  def apply(rdd: RDD[AlignmentRecord],
-            sd: SequenceDictionary,
-            rgd: RecordGroupDictionary): RDD[AlignmentRecord] = {
+  def apply(rdd: AlignmentRecordRDD): AlignmentRecordRDD = {
```

Comment:
Is it OK with you to punt this cleanup to #1081? I can make a note there to clean that up after the merge and rebase.